habitat_sim/agent/agent.py

```diff
@@ -54,18 +59,20 @@ class SixDOFPose(object):
     :property rotation: unit quaternion rotation
     """
 
-    position: np.ndarray = np.zeros(3)
+    position: np.ndarray = attr.ib(factory=_triple_zero, validator=all_is_finite)
     rotation: Union[np.quaternion, List] = np.quaternion(1, 0, 0, 0)
 
 
 @attr.s(auto_attribs=True, slots=True)
 class AgentState(object):
-    position: np.ndarray = np.zeros(3)
+    position: np.ndarray = attr.ib(factory=_triple_zero, validator=all_is_finite)
     rotation: Union[np.quaternion, List, np.ndarray] = np.quaternion(1, 0, 0, 0)
-    velocity: np.ndarray = np.zeros(3)
-    angular_velocity: np.ndarray = np.zeros(3)
-    force: np.ndarray = np.zeros(3)
-    torque: np.ndarray = np.zeros(3)
+    velocity: np.ndarray = attr.ib(factory=_triple_zero, validator=all_is_finite)
+    angular_velocity: np.ndarray = attr.ib(
+        factory=_triple_zero, validator=all_is_finite
+    )
+    force: np.ndarray = attr.ib(factory=_triple_zero, validator=all_is_finite)
+    torque: np.ndarray = attr.ib(factory=_triple_zero, validator=all_is_finite)
     sensor_states: Dict[str, SixDOFPose] = attr.Factory(dict)
```

Comment:
Can you add similar checks to `SixDOFPose`?

Reply:
@erikwijmans I do in the SixDOFPose struct itself.

Reply:
Yeah, that is just another attrs class defined right above this one

Reply:
Not sure what you are asking then?